es-app/src/MetaData.cpp

```diff
@@ -18,7 +18,8 @@ MetaDataDecl gameDecls[] = {
 	{"genre",		MD_STRING,				"unknown",			false,		"genre",				"enter game genre"},
 	{"players",		MD_INT,					"1",				false,		"players",				"enter number of players"},
 	{"playcount",	MD_INT,					"0",				true,		"play count",			"enter number of times played"},
-	{"lastplayed",	MD_TIME,				"0", 				true,		"last played",			"enter last played date"}
+	{"lastplayed",	MD_TIME,				"0", 				true,		"last played",			"enter last played date"},
+	{"favorite",	MD_INT,					"0",				true,		"favorite",				"favorite this game?"},
 };
 const std::vector<MetaDataDecl> gameMDD(gameDecls, gameDecls + sizeof(gameDecls) / sizeof(gameDecls[0]));
 
```

Comment:
Build was crashing because of an out_of_range error on launch, as it was looking for the "favorite" flag on my folders. I added 

`{"favorite", MD_INT ,"0", false},`

into folderDecls so favorite flag on folders initializes.